src/inmanta/protocol/exceptions.py

```diff
@@ -19,20 +19,27 @@
 
 from tornado import web
 
+from inmanta.types import JsonType
+
 
 class BaseHttpException(web.HTTPError):
     """
         A base exception for errors in the server
     """
 
-    def __init__(self, status_code: int = 500, message: Optional[str] = None) -> None:
+    def __init__(self, status_code: int = 500, message: Optional[str] = None, details: Optional[JsonType] = None) -> None:
         super().__init__(status_code, message)
+        self.details = details
 
     def to_body(self) -> Dict[str, Any]:
         """
             Return a response body
         """
-        return {"message": self.log_message}
+        body = {"message": self.log_message}
+        if self.details is not None:
+            body["error_details"] = self.details
```

Comment:
do we have to check if there is an existing convention?

Reply:
I checked and I was unable to find existing conventions